### PR TITLE
Move styles and use navigation-gutter style

### PR DESCRIPTION
**Pull request by AdmiralCoco**

### CODE REVIEW

File: src/pages/course-videos/d2l-capture-central-course-video-player.js [hunk 1]
```diff
@@ -75,31 +75,35 @@ class D2LCaptureCentralCourseVideoPlayer extends DependencyRequester(PageViewEle
...
 		`;
 	}
 
 	render() {
-		if (this._loading) {
-			return html`<d2l-loading-spinner size=150></d2l-loading-spinner>`;
-		}
+		const { langterm, previousLocation } = this._navigationInfo;
+		return html`
+			<div class="d2l-capture-central-course-video-player d2l-navigation-gutters">
```
Comment (qwangd2l) [inline src/pages/course-videos/d2l-capture-central-course-video-player.js:90]: Is the class `d2l-capture-central-course-video-player` defined? Or is it just left over from a previous iteration?

File: src/pages/presentations/d2l-capture-central-presentations.js [hunk 2]
```diff
@@ -35,12 +35,13 @@ class D2lCapturePresentations extends contentSearchMixin(DependencyRequester(Pag
...
 	}
 	static get styles() {
-		return [ d2lTableStyles, heading2Styles, sharedManageStyles, sharedTableStyles, css`
+		return [ d2lTableStyles, heading2Styles, sharedManageStyles, sharedTableStyles, navigationSharedStyle, css`
 		`];
 	}
 
 	constructor() {
 		super();
+		this.apiClient = this.requestDependency('content-service-client');
```
Comment (qwangd2l) [inline src/pages/presentations/d2l-capture-central-presentations.js:44]: Duplicate requestDependency call (line 46 also sets this)

File: src/util/content-service-client.js [hunk 3]
```diff
@@ -209,7 +209,7 @@ export default class ContentServiceClient {
 		});
 	}
 
-	deleteContent(contentId) {
+	deleteContent({ contentId }) {
```
Comment (qwangd2l) [inline src/util/content-service-client.js:212]: Do we need to change the call to deleteContent here?:
capture-central/src/pages/presentations/d2l-capture-central-presentations.js:
https://github.com/Brightspace/capture-central/blob/55e0a977b0ed7ff40c47d128153a90a86fab4302/src/pages/presentations/d2l-capture-central-presentations.js#L89

File: src/pages/course-videos/d2l-capture-central-course-video-player.js (hunk 1 above)
Comment (AdmiralCoco) [reply src/pages/course-videos/d2l-capture-central-course-video-player.js:90]: Oh yeah this is unnecessary, just added it as part of the wrapping div

File: src/pages/presentations/d2l-capture-central-presentations.js (hunk 2 above)
Comment (AdmiralCoco) [reply src/pages/presentations/d2l-capture-central-presentations.js:44]: Thanks - was added accidentally from rebasing

**Review by qwangd2l — APPROVED** [review]

**Pull request merged**